docs/2.8.0/docs/daml-finance/concepts/asset-model.rst

```diff
@@ -204,10 +218,12 @@ A base account implementation is provided in
 
 The account can be created with arbitrary
 :ref:`controllers <type-daml-finance-interface-account-account-controllers-36430>`
-(for incoming and outgoing transfers).
+(for incoming and outgoing transfers). In our examples, we typically let accounts be
```

Comment:
```suggestion
(for incoming and outgoing transfers). The examples typically let accounts be
```

Reply:
changed it to: "Our examples typically let accounts be .."